### DISCUSSION TITLE
Wind Turbine Predictive Maintenance Using Machine Learning (Project 197)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#28] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project #197](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/tree/main/projects/Wind%20Turbine%20Predictive%20Maintenance%20Using%20Machine%20Learning)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by Rishabh020796** [comment]
"Hi, I am a student at IIT Indore and interested in working and collaborating on this project".

**Reply by wilsonitmo** [reply]
I am Wilson Ohwedor, a master's degree student at saint Petersburg's National research University of ITMO. Russian federation. I am interested in building on this project and possible collaboration with researchers in sustainable energy systems.

**Comment by mukulanand27** [comment]
Hi, I am a student at Jadavpur University and interested in working and collaborating on this project

**Reply by robertogl** [reply]
Hello @Rishabh020796 and @mukulanand27. Great to see you will be working on this project! Do not hesitate to ask questions or write comments about the project in this discussion.

**Comment by mukulanand27** [comment]
sure we can proceed.

On Wed, Nov 10, 2021 at 4:10 AM wilsonitmo ***@***.***> wrote:

> I am Wilson Ohwedor, a master's degree student at saint Petersburg's
> National research University of ITMO. Russian federation. I am interested
> in building on this project and possible collaboration with researchers in
> sustainable energy systems.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/mathworks/MathWorks-Excellence-in-Innovation/discussions/28#discussioncomment-1615782>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUM4WPNAKUZ2SOUDB4VR4YTULGPOTANCNFSM42CFOVSQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Mukul Anand
NDF Research Scholar
Electrical Engineering Department
Jadavpur University, Kolkata

**Comment by oriabure** [comment]
Hi, I'm a student of TUDLIN. I'm interested in collaborating on this project

**Comment by mukulanand27** [comment]
Sure then tell me how we can proceed


On Monday, January 31, 2022, oriabure ***@***.***> wrote:

> Hi, I'm a student of TUDLIN. I'm interested in collaborating on this
> project
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/mathworks/MathWorks-Excellence-in-Innovation/discussions/28#discussioncomment-2078244>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUM4WPJII5ILBVVVJSFS3MLUYXAX7ANCNFSM42CFOVSQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you were mentioned.Message ID:
> <mathworks/MathWorks-Excellence-in-Innovation/repo-
> ***@***.***>
>


-- 
Mukul Anand
NDF Research Scholar
Electrical Engineering Department
Jadavpur University, Kolkata

**Reply by oriabure** [reply]
Please I am interested. I am trying to create a wind turbine model using a Simscape driveline. Any idea will be helpful. I could only archive that using Simulink.

**Reply by robertogl** [reply]
Hi @oriabure,

Have you looked at this [wind turbine model](https://www.mathworks.com/matlabcentral/fileexchange/25752-wind-turbine-model)?

**Reply by oriabure** [reply]
I have looked at it infact its great. Please  Im having difficulty in configuring the model to run.  It does have all the components needed. But I'm not able to configure it to work. Please is there an instruction for configuring the whole system? Please you response will be greatly appreciated. Thanks

**Reply by robertogl** [reply]
The model's [GitHub repository](https://github.com/mathworks/Simscape-Wind-Turbine) has reports and documentation. did you go through those already?

**Reply by oriabure** [reply]
Yes I did.

**Reply by robertogl** [reply]
I am sorry to hear that. This [video tutorial](https://www.mathworks.com/videos/model-based-design-of-a-wind-turbine-81625.html) may give some more insights. However, if you could give me more details regarding your issues I can try to better help.

**Comment by oriabure** [comment]
Thank very much your video is quite helpful.
However, I am trying to establish a link between the model and the requirement document. I started by running the project file. Attached is the result I got. I attempted using both the online and desktop version of matlab. I got same result. I am still trying to trouble shoot the system because I would really want to see it work for me. The video is great
Below are some screen shots. I will appreciate your suggestion Thanks.



Osemudiamen Oriabure
Engineering Student

BEng. Year 4
TU Dublin, City Campus, Grangegorman

Email: ***@***.***
Mobile: 0871877695
________________________________

**Reply by robertogl** [reply]
Unfortunately I do not see any attachment. Can you try again? Thanks.

**Comment by iFranzine** [comment]
Hi, I am a student at THM Friedberg in Germany and I am interested in working on this project.

**Comment by mukulanand27** [comment]
Sure tell me how can I help you?

**Comment by mukulanand27** [comment]
Send me the model so that i can see.

**Comment by clethinks** [comment]
Hello there, anyone willing to collaborate on this project?

**Reply by ciandoyle251** [reply]
yeah im free to collaborate if you would like

im currently working on the project right now

**Reply by clethinks** [reply]
Thought no one could have responded to this. 
Please share your email

**Reply by ciandoyle251** [reply]
hey sorry for the late reply

heres my email

doylec62@tcd.ie

**Comment by mukulanand27** [comment]
***@***.***

**Comment by LaurentMorrtie** [comment]
Dear teacher, we are a group of college students who hope to do some work in this project. We tried to export data by model, but encountered some problems. Because the model was rigid, the exported data seemed to be the same, so we could not make prediction, so we would like to ask for your help.

**Reply by robertogl** [reply]
Hi @LaurentMorrtie,

Could you please provide more details?

**Reply by LaurentMorrtie** [reply]
Hi，teacher.We can't get correct data from the model.We need vibration,but we can't get it.

**Reply by robertogl** [reply]
Hi @LaurentMorrtie,

You need to inject faults into your model to generate the data that you need. Some Simulink blocks already have built-in options to add faults to simulate a faulty system. Otherwise you would have to slightly modify the model. Have you done that already?

**Reply by LaurentMorrtie** [reply]
Thank you for your reply，teacher.@robertogl 
Yes, teacher. From the bad results, we forgot that we didn't inject the fault into the model. And we want to know how we can do that? We tried for a long time, but we couldn't get it right
